CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
     - Use auto constructed containers for list and map members. [#497](https://github.com/aws/aws-sdk-java-v2/pull/497), [#529](https://github.com/aws/aws-sdk-java-v2/pull/529), [#600](https://github.com/aws/aws-sdk-java-v2/pull/600)
     - Various AsyncClient Refactors:\n - Drop async prefix in `SdkAyncClientBuilder`: `SdkAsyncClientBuilder.asyncHttpClientBuilder() -> SdkAsyncClientBuilder.httpClientBuilder()`\n - Create `SdkEventLoopGroup` to allow users to provide `EventLoopGroup` and `ChannelFactory`.
     - upgrade Netty 4.1.22.Final to Netty 4.1.26.Final
+    - Allow clients to set Socket Channel Option
```

Comment:
Sorry, @zoewangg meant to add a new entry by using the [`scripts/new-change`](https://github.com/aws/aws-sdk-java-v2/blob/master/scripts/new-change) script to add new entry in the changelog queue (see the [`.changes`](https://github.com/aws/aws-sdk-java-v2/tree/master/.changes) directory). This file is autogenerated and will be populated with your addition when we do the next release.